src/components/welcome/Header.js

```diff
@@ -17,3 +31,9 @@ export default class Header extends Component {
     )
   }
 }
+
+const mapStateToProps = state => ({
+  gh: state.gh
+});
+
+export default connect(mapStateToProps, { loading, loaded })(withRouter(Header));
```

Comment:
create a variable for dispatch props

Reply:
@yashLadha this is just the header component of the welcome page, this is not even the real header, and I just put the state and actions to demonstrate the working of redux, do you still want me to do that?

Reply:
See it's not just to get things to work done but also make things that can be maintainable later as well. If we make a dispatch variable and later on we need to extend or some newcomer needs to extend the functionality then he gets a clear idea of where the functions are defined, and also that would be a good practice too.